python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -277,7 +279,40 @@ def _initialize_js(self):
             ExplanationDashboard._dashboard_js = f.read()
 
 
+def _serialize_json_safe(o):
```

Comment:
If this is copied code we should either deduplicate and keep this in a common location or also copy the unit tests for this duplicated block in case they diverge

Reply:
will do this in a follow-up PR, maybe we can move it into some common utilities file that interpret-community and azureml-interpret can re-use